scripts/restaurant_ratings_group.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+'''
+Script to parse through all yelp users and create a mapping
+from restaurant_id => [user ids, ratings] for
+easier access. Writes to 'restaurants' file as json
+'''
+
+
+import json
+import os
+from itertools import islice
+
+'''
+dict to hold restaurants mapping to user reviews:
+{
+    restaurnt_id => [
+        (user_id, rating), ...
+    ]
+    ...
+}
+'''
+restaurants = {}
+
+def main():
+    yelp_user_data_dir = os.getcwd() + "/../data/yelp_users/"
```

Comment:
Is there a reason you don't want to just run this from the directory above instead of including the ../ every time?